packages/ui/Scrollable/Scrollable.js

```diff
@@ -857,6 +857,38 @@ class ScrollableBase extends Component {
 		}
 	}
 
+	getReachedEdgeInfo = () => {
+		const
+			bounds = this.getScrollBounds(),
+			reachedEdgeInfo = {bottom: false, left: false, right: false, top: false};
+
+		if (this.canScrollHorizontally(bounds)) {
+			const
+				rtl = this.state.rtl,
+				edge = this.getEdgeFromPosition(this.scrollLeft, bounds.maxLeft);
+
+			if (edge) { // if edge is null, no need to check which edge is reached.
+				if ((edge === 'before') === rtl) {
```

Comment:
I like the following code instead of the code above.
`rtl && edge !== 'before' || !rtl && edge === 'before'`